api/service/legacysync/syncing.go

```diff
@@ -1029,7 +1033,7 @@ func (ss *StateSync) RegisterNodeInfo() int {
 
 // getMaxPeerHeight gets the maximum blockchain heights from peers
 func (ss *StateSync) getMaxPeerHeight(isBeacon bool) uint64 {
-	maxHeight := uint64(0)
+	maxHeight := uint64(math.MaxUint64)
```

Comment:
actually, seems the change won't work:

if response != nil && maxHeight < response.BlockHeight {
				maxHeight = response.BlockHeight
			}

Reply:
Good point. Fixed and thanks. Please review again.